controllers/gce/main.go

```diff
@@ -183,20 +186,23 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
-	var config *restclient.Config
+	var config *rest.Config
 	// Create kubeclient
 	if *inCluster {
-		if config, err = restclient.InClusterConfig(); err != nil {
+		if config, err = rest.InClusterConfig(); err != nil {
 			glog.Fatalf("error creating client configuration: %v", err)
 		}
 	} else {
-		config, err = clientConfig.ClientConfig()
+		config, err = clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+			&clientcmd.ClientConfigLoadingRules{},
+			&clientcmd.ConfigOverrides{}).ClientConfig()
```

Comment:
Now that `clientcmd.ClusterDefault` is being deprecated, I assume we should either specify the master address via flag or assume in-cluster. I think we can make this change - will need to remember this when bumping the manifest file.